test/rules/shopify-leads-test.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const loadRule = require('../utils/load-rule');
+const ContextBuilder = require('../utils/contextBuilder');
+const RequestBuilder = require('../utils/requestBuilder');
+
+const ruleName = 'add-country';
```

Comment:
This isn't actually executing your rule, it's executing the `add-country` rule instead.

Reply:
Good point, I'm being too hasty.

Reply:
Repairing this now with new test primarily written by @twistedstream